### DISCUSSION TITLE
New applications

### DISCUSSION THREAD

**Discussion by defdefred** [VendelinSlezak/BleskOS#41] [Ideas]
I can imagine few applications that will be useful for bleskOS users:
- calculator (simple by default + different flavor: ip mode, developper, science, finance, calendar...)
- agenda/todo list
- spreadsheet (with graghing mode + saving graphic capability). You already talk about it...
- screen-shooter (to ease, making some documentation, maybe started from main windows with a delay to switch to the desired program)

I can imagine too some applications for advance user/education, that could be compiled only if a #define is set:
- interpreted scripting language with graphics capabilities.

I can imagine too some applications for contributor user, that could be compiled only if a #define is set:
- performance analyzer (to validate optimization)
- font builder (fast bitmap and nice scalable for hidpi screen)

I'm motivated by the performance analyzer, if you are ok with contribution.

**Comment by VendelinSlezak** [comment]
Great! All of those are nice ideas. If you want to write performance analyzer you are welcome. It would fit in System board. From other programs I can add wrote quite fast some basic version of Calculator and Screen-shooter. Also from longer point of time, I am currently designing Document viewer that will be capable of working with DOCX and ODT files and then I will go on Spreadsheet editor that will be capable of working with XLSX and ODS files.

**Reply by defdefred** [reply]
Great project!
Lot's of knowledge to acquire...
